ibis/backends/bigquery/registry.py

```diff
@@ -785,8 +785,10 @@ def _timestamp_delta(t, op):
     ops.BitwiseXor: lambda t, op: f"{t.translate(op.left)} ^ {t.translate(op.right)}",
     ops.BitwiseOr: lambda t, op: f"{t.translate(op.left)} | {t.translate(op.right)}",
     ops.BitwiseAnd: lambda t, op: f"{t.translate(op.left)} & {t.translate(op.right)}",
-    ops.BitwiseLeftShift: lambda t, op: f"{t.translate(op.left)} << {t.translate(op.right)}",
-    ops.BitwiseRightShift: lambda t, op: f"{t.translate(op.left)} >> {t.translate(op.right)}",
+    ops.BitwiseLeftShift: lambda t,
+    op: f"{t.translate(op.left)} << {t.translate(op.right)}",
+    ops.BitwiseRightShift: lambda t,
+    op: f"{t.translate(op.left)} >> {t.translate(op.right)}",
```

Comment:
Will raise an issue for these.

Reply:
Looks like it's known/being worked on: https://github.com/astral-sh/ruff/issues/8179